frontend/src/components/Counters/Counters.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { useEffect, useState, useRef } from "react";
+
+//project resources
+import { getChainsLength } from "../../util/firebase/chain";
+import SingleCounter from "./SingleCounter";
+
+//mui
+import { makeStyles } from "@material-ui/core";
+import theme from "../../util/theme";
+import { number } from "yup/lib/locale";
+import ArrowDownwardIcon from "@mui/icons-material/ArrowDownward";
+
+const Counters = () => {
+  const classes = makeStyles(theme as any)();
+  const containerRef = useRef(null);
+
+  const [chains, setChains] = useState(0);
+
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      const chainsLengthResponse = await getChainsLength();
```

Comment:
This retrieves all the chains from the firestore only to compute the length. Bit inefficient, given the chain data is available in other parts of the application.

In the future you guys might consider retrieving the chains once, and then sharing that data through a react context.

Reply:
Makes sense - created a new issue for this  #136